emoji/emoji_transfer_service.py

```diff
@@ -26,21 +26,11 @@ def collision_free_name(destination_dict, source_emoji_name, prefix="cav"):
 
     return destination_name
 
-def transfer(source_client, destination_client, source_emoji_name):
+def transfer(source_dict, destination_client, source_emoji_name):
```

Comment:
i think https://github.com/gmkohler/emoji-exodus/blob/9484a99d020abd0c06edc265a6e306e889cab29f/application.py#L11 also needs to be updated to match this change in api

Reply:
good catch